packages/soundxyz/src/Soundxyz.test.ts

```diff
@@ -201,3 +204,14 @@ describe('simulateMint function', () => {
     expect(request.value).toBe(value)
   })
 })
+
+describe('getExternalLink', () => {
+  test('should return the correct link', async () => {
+    const contractAddress: Address =
+      '0xE39Df1AD806e84de47D0F6ddf56a0007C678597c'
+    const mintParams = { contractAddress, chainId: Chains.BASE }
+
+    const link = await getExternalUrl(mintParams)
+    expect(link).equals('https://www.sound.xyz/33below/midnight-diner-ii')
```

Comment:
Is there a referral query param for Sound?

Reply:
There is. Good call. Will update asap